react/features/toolbox/functions.web.js

```diff
@@ -53,16 +53,23 @@ export function getToolboxHeight() {
  *
  * @param {string} name - The name of the setting section as defined in
  * interface_config.js.
+ * @param {Object} state - The redux state.
  * @returns {boolean|undefined} - True to indicate that the given toolbar button
- * is enabled, false - otherwise. In cases where interfaceConfig is not available
+ * is enabled, false - otherwise. In cases where state and interfaceConfig are not available
  * undefined is returned.
  */
-export function isButtonEnabled(name: string) {
-    if (typeof interfaceConfig === 'object' && Array.isArray(interfaceConfig.TOOLBAR_BUTTONS)) {
-        return interfaceConfig.TOOLBAR_BUTTONS.indexOf(name) !== -1;
+export function isButtonEnabled(name: string, state: Object) {
```

Comment:
I'd like us to default to all buttons being enabled if the option is undefined (new default value).